server/templates/static/faq.html

```diff
@@ -43,138 +43,107 @@ <h1>Frequently Asked Questions </h1>
     <dd>There is no cost for the publicly available data, which is hosted on
       Google Cloud by Data Commons. For individuals or organizations who exceed
       the free usage limits, pricing will be in line with the <a
-      href="https://cloud.google.com/bigquery/public-data">BigQuery public dataset
-      program</a>.
+        href="https://cloud.google.com/bigquery/public-data">BigQuery public dataset program</a>.
+
+    <dt>Q: What is the difference between Data Commons and other other public dataset projects?
+    <dd>Many public dataset projects provide a great service by aggregating
+      topical open data sets. However, using those data sets to answer specific
+      questions often involves 'foraging' — finding the data, cleaning the data,
+      reconciling different formats and schemas, figuring out how to merge data
+      about the same entity from different sources, etc. This error-prone and
+      tedious process is repeated, once (or more) by each organization working on an issue. This is a challenge in almost every area of study involving data, from the social sciences and physical sciences to public policy. Data Commons does this work once, on a large scale, and provide cloud-accessible APIs to the cleaned, normalized and joined data. While there are millions of datasets in every domain, some collections of data get used more frequently than others.
```

Comment:
Data Commons does this work on a large scale and PROVIDES (not provide) right?

Reply:
Yup, good catch, thanks.